CHANGELOG

```diff
@@ -1,3 +1,96 @@
+== OpenDataPlane (1.45.1.0)
+
+=== Backward compatible API changes
+==== Atomic
+* Add new `odp_atomic_fetch_min_u32()` and `odp_atomic_fetch_min_u64()`
+functions for fetching and updating minimum values of atomic `uint32_t` or
+`uint64_t` variables.
+* Add new `odp_atomic_fetch_max_u32()` and `odp_atomic_fetch_max_u64()`
+functions for fetching and updating maximum values of atomic `uint32_t` or
+`uint64_t` variables.
+* Add functions for atomically setting and clearing bits in 32-bit atomic
+variables: `odp_atomic_bit_set_u32()`, `odp_atomic_bit_fetch_set_u32()`,
+`odp_atomic_bit_clr_u32()`, `odp_atomic_bit_fetch_clr_u32()`,
+`odp_atomic_bit_set_rel_u32()`, `odp_atomic_bit_clr_rel_u32()`
+* Add functions for atomically setting and clearing bits in 64-bit atomic
+variables:  `odp_atomic_bit_set_u64()`, `odp_atomic_bit_fetch_set_u64()`,
+`odp_atomic_bit_clr_u64()`, `odp_atomic_bit_fetch_clr_u64()`,
+`odp_atomic_bit_set_rel_u64()`, `odp_atomic_bit_clr_rel_u64()`
+* Add flags for the new atomic functions into `odp_atomic_op_t`.
+
+==== CPU
+* Add instruction prefetch function `odp_prefetch_l1i()`.
+
+==== DMA
+* Add debug function `odp_dma_transfer_id_to_u64()` for getting a printable
+value for a DMA transfer identifier.
+
+==== Init
+* Add debug function `odp_instance_to_u64()` for getting a printable value for
+an ODP instance handle.
+
+==== ML
+* Add functions for supporting quantization of `int16_t` and `uint16_t` data
+types: `odp_ml_fp32_to_uint16()`, `odp_ml_fp32_from_uint16()`,
+`odp_ml_fp32_to_int16()`, `odp_ml_fp32_from_int16()`
+
+==== Packet
+* Add debug function `odp_packet_tx_compl_to_u64()` for getting a printable
+value for a Tx completion handle.
+* Clarify `odp_packet_reset_max_len()` documentation to state that the returned
+length may be less than the current packet data length if data length has been
+increased by utilizing head- or tailroom.
+
+==== Scheduler
+* Add debug function `odp_schedule_group_to_u64()` for getting a printable
+value for a schedule group handle.
+
+==== Std
+* Clarify `odp_bool_t` documentation by stating how the data type can by used
+with standard C booleans and what is considered true/false.
+* Add a field of type `\__uint128_t` in the union in `odp_u128_t` when the
```

Comment:
Why the backslash?

Reply:
Without it `__` is interpreted as italic formatting in the output file.
E.g.
![Screenshot 2024-09-03 at 10 27 48](https://github.com/user-attachments/assets/3dc9e3a2-1711-4922-aa06-f3c41a8908b6)

Reply:
There's another occurrence on the next line without a backslash.